account-decoder/src/parse_token.rs

```diff
@@ -102,8 +109,8 @@ pub struct UiTokenAccount {
     pub delegate: Option<String>,
     pub is_initialized: bool,
     pub is_native: bool,
-    #[serde(skip_serializing_if = "UiTokenAmount::is_zero")]
-    pub delegated_amount: UiTokenAmount,
+    #[serde(skip_serializing_if = "Option::is_none")]
+    pub delegated_amount: Option<UiTokenAmount>,
```

Comment:
Without the Option, you can't deserialize a `UiTokenAccount` in Rust.  Ran into this in the spl-token-cli